test/coreneuron/test_pointer.py

```diff
@@ -1,3 +1,5 @@
+import os
+import distutils.util
```

Comment:
x-ref: https://github.com/neuronsimulator/nrn/pull/2193

Reply:
I assume we will addressed this in #2193, right?